Tests/PostgresNIOTests/New/Messages/NotificationResponseTests.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  File.swift
+//
+//
+//  Created by Fabian Fett on 07.01.21.
+//
+
+import NIO
```

Comment:
Ditch Xcode template headers.
```suggestion
import NIO
```